modulus/models/module.py

```diff
@@ -65,7 +65,7 @@ def __new__(cls, *args, **kwargs):
         out._args = {
             "__name__": cls.__name__,
             "__module__": cls.__module__,
-            "__args__": {k: v for k, v in bound_args.arguments.items()},
+            "__args__": {k: v for k, v in bound_args.kwargs.items()},
```

Comment:
I'm not sure this is the right solution here. This won't cover any positional arguments, will it?

The root cause for this issue makes sense since the docs say that `arguments` will only have explicitly bound arguments: https://docs.python.org/3/library/inspect.html#inspect.BoundArguments.arguments

But `kwargs` only has the keyword args and `args` has positional arguments. The current solution will skip any positional args.

Reply:
The positional arguments get bound to the kwargs from the above two lines. Let me take a look though.

Reply:
Oh weird,
```python
        print(bound_args.kwargs)
        bound_args.arguments.pop("self", None)
        print(bound_args.kwargs)
        exit()
```
This gives,
```
{'boop': 2, 'beep': 4}
{'in_channels': 1, 'out_channels': 3, 'boop': 2, 'beep': 4}
```
popping `self` causes kwargs to get all the arguments? What is going on haha.

Reply:
Dang, I am not sure a good way to fix this...